languages/python/oso/oso/oso.py

```diff
@@ -42,6 +42,9 @@ def is_allowed(self, actor, action, resource) -> bool:
         except StopIteration:
             return False
 
+    def enable_roles(self):
+        self.load_file("oso/roles.polar")
```

Comment:
Reminder that we will need to modify the `setup.py` to include this as an additional file in the package.